components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt

```diff
@@ -68,6 +68,15 @@ interface HistoryStorage : Storage {
         excludeTypes: List<VisitType> = listOf()
     ): List<VisitInfo>
 
+    /**
+     * Returns a list of the top frecent site infos limited by the given number of items
+     * sorted by most to least frecent.
+     *
+     * @param numItems the number of top frecent sites to return in the list.
+     * @return a list of the top frecent site infos sorted by most to least frecent.
```

Comment:
nit: `a list of [TopFrecentSiteInfo], most frecent first.`

Reply:
Fixed.